src/azure-cli/azure/cli/command_modules/appservice/_constants.py

```diff
@@ -39,39 +39,18 @@
         'java': '8'
     }
 }
-# functions version -> runtime -> runtime version : container image
-RUNTIME_TO_IMAGE_FUNCTIONAPP = {
+# functions version -> runtime : runtime versions
+RUNTIME_TO_SUPPORTED_VERSIONS_FUNCTIONAPP = {
```

Comment:
just to confirm, container image supporting is replaced with supported version of different runtime?

Reply:
Yeah the only difference is we don't need to specify the container image anymore so I renamed this because "Image" no longer applies. We still need to track which `runtime_version`'s we support for which runtime and functions version.

Reply:
Hi @gzuber, to keep the language consistent, would it make sense to rename this to something like FUNCTIONSVERSION_TO_SUPPORTED_RUNTIMEVERSION given that you take into consideration the `Host version (function version)` ?

Reply:
Yeah that's a good point, I didn't need to hold on to the old names. I've changed a lot of the constants to better reflect what they are.